keyboards/ealdin/quadrant/info.json

```diff
@@ -0,0 +1,25 @@
+{
+    "keyboard_name": "Quadrant E-C",
+    "keyboard_folder": "quadrant",
+    "url": "https://github.com/Ealdin/keyboards/tree/master/Quadrant", 
+    "maintainer": "Ealdin Keyboards", 
+    "width": 14, 
+    "height": 5, 
+    "layouts": {
+        "LAYOUT_Quadrant_5x14": {
```

Comment:
None of these match the layout macros in quadrant.h

Reply:
Sorry, changed them on the quadrant.c file and forgot this. Fixed now.